projects/Reth_contributor.md

```diff
@@ -0,0 +1,104 @@
+# Project Proposal
+
+Become a core contributor for [Reth](https://github.com/paradigmxyz/reth), a new Ethereum execution client written in Rust!
+
+## Motivation
+
+Client diversity is one of the most important features of Ethereum as it helps to further decentralise the network. Right now there are already several implementations for the Ethereum protocol, divided in consensus and execution clients, as you can see from the image below.
+
+![client_diversity.png](Reth_contributor/client_diversity.png)
```

Comment:
I would just add link to the client diversity website and/or eth org docs https://ethereum.org/en/developers/docs/nodes-and-clients/client-diversity/